index.js

```diff
@@ -43,23 +43,35 @@ function getNavigationFromProject(baseUrl = '', project) {
   return nav ?? []
 }
 
-export async function generateApiDocs({
+const typedocConfig = {
+  excludeInternal: true,
+  excludePrivate: true,
+  excludeProtected: true,
+  githubPages: false
+}
+
+const markdownPluginConfig = {
+  hideBreadcrumbs: true,
+  hideInPageTOC: true,
+  hidePageHeader: true,
+  hidePageTitle: true
+}
+
+const removeTrailingSlash = (pathString = '') =>
```

Comment:
You are mixing `function a()` and `const a = () =>` syntax in the same file. Let’s choose one.